### DISCUSSION TITLE
Change Default page

### DISCUSSION THREAD

**Discussion by jdenos** [YOURLS/YOURLS#3058] [Q&A] [answered]
Hello. I love yourls after installing it everything work fine. 
I setup my go.domain.com to serve as the site and every short link works well. the go.domain.com/admin leads me to panel. 

How ever if I go to the site go.domain.com I get a 403 error! I would like to redirect it to my main website. Is it possible ?

Thanks in advance!

**Comment by dgw** [comment, marked as answer]
Very simple `index.php` will do it assuming your HTTP server is set up to treat such a file as the directory index (like `index.html` is by default).

```php
<?php
header("HTTP/1.1 301 Moved Permanently"); 
header("Location: https://your.main.site/");
```

**Reply by jdenos** [reply]
many thanks
